styles/layout.less

```diff
@@ -528,6 +528,11 @@ div.newsitem {
     img {
         max-width: 100%;
     }
+    iframe {
+      width: 320px;
+      height: 205px;
+      border: 0;
+    }
```

Comment:
I've unarchived the news item for the PP page that has graphs in it. Not all of the graphs are the same size, and in the case of the PP one (and the AH one on PROD), the second graph really shouldn't be the same size as the other one, so if this has to be specified here, it should probably have some specificity added..

<img width="457" alt="image" src="https://user-images.githubusercontent.com/36800239/124001963-d3690c00-d99a-11eb-9a2c-c4f716a9560e.png">

Based on other places where there are multiple graphs, there should probably be some king of visual separation, also.

Reply:
I think this is something that the news item needs to define based on the graph they want to include. I wouldn't add it to the global CSS.